sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionInfo.java

```diff
@@ -150,4 +150,12 @@ public ExpressionInfo(String className, String db, String name, String usage, St
         // simply pass the `extended` as `arguments` and an empty string for `examples`.
         this(className, db, name, usage, extended, "", "", "", "");
     }
+
```

Comment:
Do we really need to move this function and change the getters in ```ExpressionInfo.java```?

Reply:
I guess this is OK. @juliuszsompolski, what do you think about calling the ```replaceFunctionName``` directly inside the getters?

Reply:
I think it's a move in a good direction, as it will make it work better (not return nulls, return actual name instead of placeholder) for anyone using ExpressionInfos directly through `SessionCatalog.lookupFunctionInfo` API directly
cc @gatorsmile, what do you think?

Reply:
Create new PR for this change: https://github.com/apache/spark/pull/25314

Reply:
It's merged. Please rebase this PR. Thanks!

Reply:
Done